conda/rapids-tpcx-bb.yml

```diff
@@ -25,6 +25,8 @@ dependencies:
   - psutil
   - ipykernel
   - jupyterlab
+  - gspread
```

Comment:
We probably don't want to enforce this yet, but I don't think it's a blocker.

We aren't set up to fail gracefully if `gspread` is present but the GOOGLE_SHEETS_CREDENTIALS_PATH is not set.

Reply:
Ah, I see you've wrapped the logic. Great